packages/utils/lib/axios.ts

```diff
@@ -20,6 +20,7 @@ if (hasHttpsProxy) {
 }
 
 export const axiosInstance = axios.create({
+    proxy: false,
```

Comment:
this will disable the default axios proxy, and work with set httpProxyAgent, currently default axios proxy takes precedence and causes issue as it finds a set https_proxy env

Reply:
done